src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -1229,9 +1229,7 @@ class spell_pal_blinding_light : public SpellScriptLoader
                 if (Player* _player = GetCaster()->ToPlayer())
                 {
```

Comment:
You can leave these away too. I think its more common to remove all brackets besides the very inner brackets.